### ISSUE TITLE
Please add checks for perl-brace-format (Locale::TextDomain)

### ISSUE THREAD

**Issue by jwilk** [jwilk/i18nspector#6]
[GNU gettext documentation](https://www.gnu.org/software/gettext/manual/html_node/perl_002dformat.html) says:
> Perl brace format strings are described in the [Locale::TextDomain](https://metacpan.org/pod/Locale::TextDomain) manual page […]. In brief, Perl format uses placeholders put between braces (`{` and `}`). The placeholder must have the syntax of simple identifiers.

Example PO snippet:
```po
#: ../clawsker:2267 
#, perl-brace-format
msgid "Perl-GTK3 version {gtkv}, built for {gtkb}, running with {gtkr}."
msgstr "Perl-GTK3 versió {gtkv}, muntatge per a {gtkb}, s'executa amb {gtkr}."
```

In #5, @intrigeri suggested adding support for this format.

The format is extremely simple, so this should be straight-forward to implement.

**Comment by jwilk** [comment]
Fixed in [0.26](https://github.com/jwilk/i18nspector/releases/tag/0.26).

**Comment by intrigeri** [comment]
Thanks a lot! :)
